checkout_externals

```diff
@@ -17,8 +17,8 @@ import manic
 
 if sys.hexversion < 0x02070000:
     print(70 * '*')
-    print('ERROR: {0} requires python >= 2.7.x. '.format(sys.argv[0]))
-    print('It appears that you are running python {0}'.format(
+    print(f'ERROR: {0} requires python >= 2.7.x. '.format(sys.argv[0]))
+    print(f'It appears that you are running python {0}'.format(
```

Comment:
If we are removing python2, why is the check still for python 2.7 and not 3?

Reply:
Fixed. thank you.